yt/frontends/swift/data_structures.py

```diff
@@ -15,7 +16,7 @@ class SwiftParticleFile(ParticleFile):
 class SwiftDataset(SPHDataset):
     _load_requirements = ["h5py"]
     _index_class = SPHParticleIndex
-    _field_info_class = SPHFieldInfo
+    _field_info_class = SwiftFieldInfo
```

Comment:
not clear to me what are the implications of this change in terms of backward compatibility

Reply:
Anyone want to comment here ? For the record, I don't think it's critical enough to hold back the PR, but I'd like an answer if possible.

Reply:
Someone with more of an expertise in swift data might want to chime in, but I just spent a while walking through the initialization and aliasing that happens and I don't see any issues. It will still check for all the fields defined by `SPHFieldInfo`, and the additional aliasing that `SwiftFieldInfo` adds will only happen if the new fields exist within the dataset.